components/renku_data_services/crc/orm.py

```diff
@@ -133,6 +136,24 @@ def dump(self, matching_criteria: models.ResourceClass | None = None) -> models.
         )
 
 
+class ClusterORM(BaseORM):
+    """Cluster definition."""
+
+    __tablename__ = "clusters"
+    id: Mapped[ULID] = mapped_column("id", ULIDType, primary_key=True, default_factory=lambda: str(ULID()), init=False)
+    name: Mapped[str] = mapped_column(String(40), unique=True, index=True)
+    config_name: Mapped[str] = mapped_column(String(40), unique=True, index=True)
+
+    def to_cluster(self) -> SavedCluster:
+        """Create a cluster model from the ORM object."""
+        return SavedCluster(id=self.id, name=self.name, config_name=self.config_name)
+
+    @classmethod
+    def from_model(cls, cluster: Cluster) -> "ClusterORM":
```

Comment:
nitpick: In all other instances we call these methods dump and load.

Reply:
renamed